wave_matching/tests/ndt_tests.cpp

```diff
@@ -53,7 +53,7 @@ TEST_F(NDTTest, fullResNullMatch) {
     match_success = matcher->match();
     double diff = (matcher->getResult().matrix() - perturb.matrix()).norm();
     EXPECT_TRUE(match_success);
-    EXPECT_LT(diff, 0.1);
+    EXPECT_LT(diff, 0.12);
```

Comment:
If I'm not mistaken actually, are GTests Assert arguments `ASSERT(expected, actual)`?